### ISSUE TITLE
Add different resolution options for the same video, using user processing power

### ISSUE THREAD

**Issue by apenasrr** [lbryio/lbry-desktop#4288]
**Is your feature request related to a problem? Please describe.**
I opened this suggestion to go a little deeper into this [another suggestion](https://github.com/lbryio/lbry-desktop/issues/3478) made in the past by [@Tcll](https://github.com/Tcll) and to suggest in detail, a way for its application.

Many people have difficulty and are sometimes technically unable to watch a video with a higher quality than that supported by their internet speed.

**Describe the solution you'd like**
It would be interesting for lbry to facilitate the activity of re-encoding a video to different resolutions and making them easily selectable as options on the video page.

**Describe alternatives you've considered**
It would be for an addon for chrome/brave webbrowsers and/or a function in lbry_desktop that would allow you to select reencode options for other resolutions and, using some command line application (like ffmpeg) on ​​the user's machine, to reencode the video for all chosen resolutions and queue up.

Using USER MACHINE PROCESSING POWER, lbry being a mere facilitator of the reencode function.

::. TRACEABILITY of video versions with different resolutions
Therefore, a claim would be created for each version of the video with a different resolution, whose variation of the claim name would be standardized from the name of the original claim.

As an example, when selecting a 1080p video to upload, lbry.tv/desktop facilitates automatic reencode using resources from the user's machine to 'reencode in all chosen options and queue up' the videos in the claims:
Example of claims automatically created if the user chooses 2 options: 1280x720 and 858x480
-lbry://myvideooriginal
-lbry://myvideooriginal_1280_720
-lbry://myvideooriginal_858_480

::. EASE OF CHOICE for those who assist
Continuing the vision of the future, lbry.tv and lbry_desktop, when reproducing the file 'lbry://myvideooriginal', would be able to identify the existence of 2 claims pointing to different resolution versions of the same video and would suggest them on the video page original, as well as alternative videos.

This traceability in the identification may be possible by the creation of a distinct tag automatically added in the description of the videos (original and alternative videos) when using the automatic reencode function on the website or app, without needing any update in the lbry protocol. :)

In summary, this suggestion:
- Would not be required lbry organization's processing power to re-encode videos in different resolutions, using exclusively the processing power of the content creator's machine that decided to increase the quality options for its audience
- Facilitate (user-friendly) ways for users to re-encode their videos to different resolutions and make them available to their audience in a convenient way (similar to youtube)  
- It would increase the accessibility potential of videos on the lbry platform to millions of users with low speed internet

**Comment by tzarebczan** [comment]
Thanks for the issue, we have this filed under https://github.com/lbryio/lbry-desktop/issues/3878
